### PR TITLE
Test improvements

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

**Comment by dzenanz** [comment]
@thewtex please review this

File: include/itkMaxPhaseCorrelationOptimizer.h [hunk 1]
```diff
@@ -37,10 +37,6 @@ namespace itk
  *  If interpolation method is None, the shift is estimated with pixel-level
  *  precision. Otherwise the requested interpolation method is used.
  *
- * \author Jakub Bican, jakub.bican@matfyz.cz, Department of Image Processing,
- *         Institute of Information Theory and Automation,
- *         Academy of Sciences of the Czech Republic.
- *
```
Comment (thewtex) [inline include/itkMaxPhaseCorrelationOptimizer.h]: We should retain these author comments whenever possible.

Comment (dzenanz) [reply include/itkMaxPhaseCorrelationOptimizer.h]: OK, I will remove this commit.

**Comment by dzenanz** [comment]
@thewtex there are some weird Python errors in AppVeyor.

**Comment by thewtex** [comment]
> @thewtex there are some weird Python errors in AppVeyor.

> Error uploading artifact the storage: Remote server returned 503: The server is busy.

This must be on AppVeyor's infrastructure side. We could disable / remove the AppVeyor build -- it is not needed with Azure Pipelines.

**Comment by dzenanz** [comment]
This started today, so it might be some transient issue.

I recently asked Mike which data can we release, and that's what I based the new additions on. So as soon as Azure pipeline is finished, I plan to merge this.

**Pull request merged**